### PR TITLE
WIP-Create Animation-Script-Ideas-1-4-21.md

### PR DESCRIPTION
**Pull request by webworthy**
Work In Progress

Added more stuff

### CODE REVIEW

File: Animation-Script-Ideas-1-4-21.md [hunk 1]
```diff
@@ -0,0 +1 @@
+WIP coming back to start script.
```
Comment (Zaxounette) [inline Animation-Script-Ideas-1-4-21.md:1]:
```suggestion
Work In Progress coming back to start script.
```

Comment (britttttk) [inline Animation-Script-Ideas-1-4-21.md:1]:
```suggestion
WIP: Coming back to start script
Chapter 1
Chapter 2
```

**Review by britttttk — APPROVED** [review]

File: Animation-Script-Ideas-1-4-21.md (hunk 1 above)
Comment (Zaxounette) [reply Animation-Script-Ideas-1-4-21.md:1]: explicits

**Comment by britttttk** [comment]
Relates to this comment about BTCPay being a [Payment Platform](https://github.com/btcpayserver/organization/issues/12#issuecomment-755382172).

**Pull request merged**